docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -141,44 +149,148 @@ Restrictions
+of vitamin A deficiency at each time step to determine whether the simulant has
+VAD during that time step. Each simulant's propensity is assigned only once, but
+the underlying prevalence distribution can change throughout the course of the
+simulation, which may result in a change in the simulant's vitamin A status.
+
+In particular, our modeling strategy will **not** explicitly use incidence or
+remission data for vitamin A deficiency, but only *prevalence* (which is the
+same as the exposure data for the VAD risk factor). The rationale for this approach is twofold:
+
+1.  We want to guarantee that the simulated baseline prevalence of vitamin A
+    deficiency matches the prevalence data from GBD, which is likely more
+    trustworthy than incidence and remission data.
+
+2.  Relative risks from the literature about the effects of vitamin A
+    supplementation or fortification on vitamin A status are best interpreted as
+    risk ratios for prevalence of vitamin A deficiency. The exposure model
+    provides a way to directly model these effect sizes in a way that preserves
+    this interpretation.
+
+.. todo::
+
+  Verify that effect sizes on VAD should actually be interpreted as described
+  above, and that the prevalence-only model is a good way to accurately
+  represent these numbers.
+
+  Explain why the prevalence-only model is a reasonable strategy, citing
+  incidence, remission, and prevalence data, as well as expert opinions about
+  VAD. (Perhaps this explanation should come later, e.g. in the Assumptions and
+  Limitations section.)
+
+Following is a more detailed description of how the exposure model for VAD
+should work.
+
+Determining Vitamin A Status
+++++++++++++++++++++++++++++
+
+At each time step, Vivarium needs to determine whether each simulant has vitamin
+A deficiency. To do so, follow these steps:
+
+1.  **Initialize:** When simulant :math:`i` enters the simulation (e.g. at the
+    start of the simulation or at the time step when the simulant is born),
+    assign the simulant a random number :math:`v_i \sim
+    \operatorname{Uniform}([0,1])`, which we call the **VAD propensity score**
+    for simulant :math:`i`.
+
+2.  **Update:** On each time step :math:`t`:
+
+    a)  If simulant :math:`i` survives, update any of simulant :math:`i`'s
+        variables that determine which subpopulation the simulant belongs to.
+        For example, they may move into the next age group, or they may begin
+        receiving or stop receiving an intervention. Call this new subpopulation
+        :math:`\text{subpop}(i,t)`.
+
+    b)  Look up or compute the prevalence
+        :math:`p_\text{VAD}(\text{subpop}(i,t))` of vitamin A deficiency for the
+        simulant's updated subpopulation.
+
+    c)  If :math:`v_i < p_\text{VAD}(\text{subpop}(i,t))`, the simulant has
+        vitamin A deficiency on the next time step; otherwise, they don't.
+
+To address a point of potential confusion in the above algorithm, note that a
```

Comment:
Just leaving notes.  I think this section is great. I think we want to pull it out into the general risk model for the **standard way to sample from categorical distributions**.  In order to do so, we have to generalize this section a bit.  Technically what we do is start with a distribution

p = [p1, p2, ..., pN, 1 - (p1 + ... +pn)]

We then take the cumulative sum over the distribution

p_cum = [p1, p1 + p2, ..., p1 + ...+ pN, 1]

to form the right bounds of a partition of the interval [0, 1] with each subinterval mapping to a risk exposure category (and associated probability).  

We then select the category j by  arg_max_j (q_i < p_cum[j]) and assign that category as x_i.

This means the interpretation of *propensity* is dependent on the sort order of the categories.

Reply:
The default sort order is worst to best.

Reply:
w/r/t risk effect.  We have not had to deal with unordered categorical risks.